pkg/loader/loader.go

```diff
@@ -157,7 +157,7 @@ func loadOpenAPI(prg *types.Program, data []byte) *openapi3.T {
 		prg.OpenAPICache = map[string]any{}
 	}
 
-	switch isOpenAPI(data) {
```

Comment:
It should be possible to use `openapi.LoadFromBytes` (if exported) here and just add the cache part after it, right? Not sure if it's worth it for the transition period :man_shrugging: